engine/apps/alerts/escalation_snapshot/snapshot_classes/escalation_policy_snapshot.py

```diff
@@ -87,6 +90,7 @@ def __init__(
         self.custom_webhook = custom_webhook
         self.notify_schedule = notify_schedule
         self.notify_to_group = notify_to_group
+        self.notify_to_team_members = notify_to_team_members
```

Comment:
To be sure, have you checked how this works when deserializing an escalation policy snapshot where the `notify_to_team_members` field does not exist? (we should still be able to deserialize snapshots created before the field was around) Maybe we need a test for this scenario to double-check.

Reply:
I've fixed this along with adding a test for it.